extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -346,538 +355,579 @@ A **subtree file** is a binary file that contains availability information for a
-    },
-    {
-      "buffer": 0,
-      "byteOffset": 10,
-      "byteLength": 64
-    }
-  ],
-  "tileAvailability": {
-    "bufferView": 0
-  },
-  "contentAvailability": {
-    "constant": 0
-  },
-  "childSubtreeAvailability": {
-    "bufferView": 1
-  }
-}
+
+---------------------------------------
+<a name="reference-availability"></a>
+### Availability
+
+An object describing the availability of a set of elements.
+
+**`Availability` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**bufferView**|`integer`|Index of a buffer view that indicates whether each element is available. The bitstream conforms to the boolean array encoding described in the [Cesium 3D Metadata specification](../../../specification/Metadata/0.0.0/README.md). If an element is available, its bit is 1, and if it is unavailable, its bit is 0. The `bufferView` `byteOffset` must be aligned to a multiple of 8 bytes.|No|
+|**availableCount**|`integer`|A number indicating how many 1 bits exist in the availability bitstream. `availableCount` is only allowed when `bufferView` is used.|No|
+|**constant**|`integer`|Integer indicating whether all of the elements are available (1) or all are unavailable (0).|No|
+|**extensions**|`any`||No|
+|**extras**|`any`||No|
+
+Additional properties are allowed.
+
+#### availability.bufferView
+
+Index of a buffer view that indicates whether each element is available. The bitstream conforms to the boolean array encoding described in the [Cesium 3D Metadata specification](../../../specification/Metadata/0.0.0/README.md). If an element is available, its bit is 1, and if it is unavailable, its bit is 0. The `bufferView` `byteOffset` must be aligned to a multiple of 8 bytes.
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+#### availability.availableCount
+
+A number indicating how many 1 bits exist in the availability bitstream. `availableCount` is only allowed when `bufferView` is used.
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+#### availability.constant
+
+Integer indicating whether all of the elements are available (1) or all are unavailable (0).
+
+* **Type**: `integer`
+* **Required**: No
+* **Allowed values**:
+   * `0`
+   * `1`
+
+#### availability.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+#### availability.extras
+
+* **Type**: `any`
+* **Required**: No
+
+
+
+
+---------------------------------------
+<a name="reference-buffer"></a>
+### Buffer
+
+A buffer that points to a uri containing binary data.
+
+**`Buffer` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**uri**|`string`|The uri of the buffer. Relative paths are relative to the file containing the buffer JSON. If `uri` is omitted, the buffer is assumed to refer to the binary chunk of the subtree file.|No|
+|**byteLength**|`integer`|The length of the buffer in bytes.| &#10003; Yes|
+|**name**|`string`|The name of the buffer. The string is UTF-8 encoded.|No|
```

Comment:
> The string is UTF-8 encoded

Can be removed here and other places. I'm removing it from 3DTILES_metadata as well (with the exception of strings in feature table binary)